### PR TITLE
[RTM] Fix triple newlines left over from removing address comments, and other improvements

### PR DESCRIPTION
**Pull request by Rangi42**
Example:

```
Function1337: ; 1337
    ret
; 1338

Function1338: ; 1338
    ...
```

became:

```
Function1337:
    ret


Function1338:
    ...
```

so this reformats it to:

```
Function1337:
    ret

Function1338:
    ...
```

### CODE REVIEW

**Comment by mid-kid** [comment]
I actually removed the entire line if I removed the address comment on an otherwise blank line.
It's probable, however that things like
```
Function1337: ; 1337
    ret

; 1338

Function1338: ; 1338
    ...
```

became
```
Function1337: ; 1337
    ret


Function1338: ; 1338
    ...
```

I thought it was fine because there's a lot of other places where two newlines are used between functions for some reason.

File: engine/battle/ai/scoring.asm [hunk 1]
```diff
@@ -232,7 +230,6 @@ AI_Types:
 	jr .checkmove
```
Comment (mid-kid) [inline engine/battle/ai/scoring.asm]: There's still three newlines here?

File: engine/battle/ai/scoring.asm [hunk 2]
```diff
@@ -260,7 +257,6 @@ AI_Offensive:
...
```
Comment (mid-kid) [inline engine/battle/ai/scoring.asm]: Here as well? Not going to point all of them out, but there's more.

**Comment by Rangi42** [comment]
I don't know why those were missed the first time, but now they're done.

File: engine/pokemon/correct_nick_errors.asm [hunk 3]
```diff
@@ -70,5 +70,5 @@ CorrectNickErrors::
 	db "<ROUTE>",  "<GREEN>" + 1
 	db "<ENEMY>",  "<ENEMY>" + 1
 	db "<MOM>",    "<TM>"    + 1
-	db "<ROCKET>", "┘"       + 1
+	db "<ROCKET>", " "
```
Comment (yenatch) [inline engine/pokemon/correct_nick_errors.asm]: `" "` has no relation other than being one after. If border tiles don't count as control characters the problem is the labelling.

File: engine/games/card_flip.asm [hunk 4]
```diff
@@ -1,5 +1,5 @@
-CARDFLIP_LIGHT_OFF EQU $ef
-CARDFLIP_LIGHT_ON  EQU $f5
+CARDFLIP_LIGHT_OFF EQUS "\"♂\"" ; $ef
+CARDFLIP_LIGHT_ON  EQUS "\"♀\"" ; $f5
```
Comment (yenatch) [inline engine/games/card_flip.asm]: This seems like an assembler bug. `EQU "♂"` should be allowed.

Comment (Rangi42) [reply engine/games/card_flip.asm]: True; but until then the escapes are necessary, and were already used in some other files.

File: engine/pokemon/correct_nick_errors.asm (hunk 3 above)
Comment (Rangi42) [reply engine/pokemon/correct_nick_errors.asm]: I guess border tiles can count as control characters, then.

File: home/audio.asm [hunk 5]
```diff
@@ -50,8 +50,6 @@ UpdateSound::
 
 _LoadMusicByte::
 ; wCurMusicByte = [a:de]
-GLOBAL LoadMusicByte
```
Comment (yenatch) [inline home/audio.asm]: This was here because `BANK()` used to not flag labels for import. This was fixed a while ago.

**Pull request merged**